### ISSUE TITLE
Vader Immortal Episode III crash

### ISSUE THREAD

**Issue by SuicidalGamer77** [LibreVR/Revive#1393]
Whenever I attempt to start up the story, It ends up crashing and either leaving me in a black screen and/or leaving me in the steam hub with a picture saying loading. I am using intel i5 processor and a GeForce GTX 1070 and the other two episodes work fine under the conditions but only on episode three.
![problem](https://user-images.githubusercontent.com/58092650/69487846-c9257a00-0e15-11ea-9f3b-7b13aeea591e.PNG)

**Comment by TheBLKMN** [comment]
Yeah, it's leaving me stuck there too. I'm going to try former versions and see if they work.

**Comment by Kexikus** [comment]
Same for me. I always get a black screen when starting the story mode (the dojo takes a while to load but works fine). Apparently there's supposed to be an audio only summary of the previous episodes but I waited for at least three minutes or so and there was no sound at all. I could still open the menu though, so the game had not crashed.

**Comment by TheBLKMN** [comment]
Ok, I think I've made some progress.

I downloaded wolves in the walls (another relatively new Oculus game) and it gave me the same perpetual pausing issue. I can't unpause from either of them. Older Oculus titles have no issues.

Anyone solve this issue?

**Comment by 5shenGamer** [comment]
i have this issue from about a month no one find a Solution
